.pycodestyle

```diff
@@ -0,0 +1,35 @@
+# Copyright 2019 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ------------------------------------------------------------------------------
+
+[pycodestyle]
+exclude=build,docs,*_pb2.py
+#
+# For now, disable these pycodestyle checks.
+# Maintainers should endeavor to investigate and remove these for
+# maximum pycodestyle coverage.
+#
+# E126 continuation line over-indented for hanging indent
+# E128 continuation line under-indented for hanging indent
+# E722 do not use bare except
+# W191 indentation contains tabs
+# W503: line break before binary operator
+ignore=E126,E128,E722,W191,W503
```

Comment:
Except for line length, we should not omit any other rules. Let's be as close to PEP8 as possible.

Reply:
@rranjan3 I will not enable pylint on files until they are fixed.
@rranjan3 Disabling some pycodestyle rules is just temporary, because the disabled violations are very common in the current code base.

Reply:
I split this PR into two PRs. This PR and a separate PR with fixes to `tests/.../*.py`
This PR is now dependent on PR #178